src/components/data/kanban/kanban.scss

```diff
@@ -12,25 +12,29 @@
     height: 100%;
   }
 
-  &__body .mykn-column {
+  &__count {
+    margin-left: var(--spacing-h);
+    border-radius: 10px;
     border: 1px solid var(--typography-color-border);
-    overflow-y: auto;
+    background-color: #fafafa;
+    padding: 0 6px;
+    color: var(--typography-color-muted);
   }
 
-  &__body .mykn-column:before {
-    background-color: var(--typography-color-border);
-    opacity: 0.1;
-    content: "";
-    height: 100%;
-    position: absolute;
-    width: 100%;
+  &__body .mykn-column {
+    border: 1px solid var(--typography-color-border);
+    border-radius: var(--border-radius-l);
+    background-color: var(--page-color-background-alt);
+    overflow-y: auto;
+    overflow-x: hidden;
+    padding: 18px 16px;
```

Comment:
Does this make sense on all device sizes?

Reply:
Will double check